src/styles/utiles/_reset.scss

```
@@ -0,0 +1,9 @@
+*,
+*::before,
```

I suppose dat u can delete this pseudo-classes and just leave 
```suggestion
* {
box-sizing: border-box;
}
```